### PR TITLE
Increase the application start timeout value for Checkpoint

### PR DESCRIPTION
**Pull request by SmithaSubbarao**
This PR increases the application timeout issue our InstantOn FATs.

Test results: wip

### CODE REVIEW

**Comment by SmithaSubbarao** [comment]
#build

**Comment by anjumfatima90** [comment]
Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_0YKEoOk-Ee6MpPO8M5EK5Q
Instant on build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_gUG30OiLEe6_pbTBHNL0pQ

File: dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationStateCoordinator.java [hunk 1]
```diff
@@ -84,7 +85,14 @@ public static void setApplicationConfigurator(ApplicationConfigurator applicatio
     }
 
     public static void setApplicationStartTimeout(long applicationStarttimeout) {
-        startTimeout = applicationStarttimeout;
+
+        if (!CheckpointPhase.getPhase().restored()){
+	 startTimeout = applicationStarttimeout * 3;
```
Comment (tjwatson) [inline dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationStateCoordinator.java]: This line isn't following the project source formatting.  It appears this project does not have save actions set like other liberty projects that would force the formatting to be consistent on save.

File: dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationStateCoordinator.java [hunk 2]
```diff
@@ -84,7 +85,14 @@ public static void setApplicationConfigurator(ApplicationConfigurator applicatio
     }
 
     public static void setApplicationStartTimeout(long applicationStarttimeout) {
-        startTimeout = applicationStarttimeout;
+
```
Comment (tjwatson) [inline dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationStateCoordinator.java]: We want `startTimeout` to go back to the original value after restore.  To do that we should only apply 3x to the value on `getApplicationStartTimeout`

```java
    private static final CheckpointPhase checkpointPhase = CheckpointPhase.getPhase();
    public static long getApplicationStartTimeout() {
        return checkpointPhase.restored() ? startTimeout : startTimeout * 3;
    }
```

File: dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationStateCoordinator.java (hunk 1 above)
Comment (SmithaSubbarao) [reply dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationStateCoordinator.java]: Sounds good, thank you Tom! I've updated the project source formatting now, I'll address the 2nd review comment as well.

File: dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationStateCoordinator.java (hunk 2 above)
Comment (SmithaSubbarao) [reply dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationStateCoordinator.java]: Updated the remaining changes, looks like it's still running through the tests.

**Comment by SmithaSubbarao** [comment]
#build

**Comment by SmithaSubbarao** [comment]
#build

**Comment by SmithaSubbarao** [comment]
#build

**Review by tjwatson — APPROVED** [review]

**Comment by tjwatson** [comment]
#libby

**Pull request merged**